code/__DEFINES/mobs.dm

```diff
@@ -988,3 +988,8 @@ GLOBAL_LIST_INIT(layers_to_offset, list(
 #define BUTT_SPRITE_PLASMA "plasma"
 #define BUTT_SPRITE_FUZZY "fuzzy"
 #define BUTT_SPRITE_SLIME "slime"
+
+#define SHADOW_NONE ""
```

Comment:
```suggestion
#define SHADOW_NONE "none"
```
If we're gonna make these macros we should ideally not rely on falsey/truthy as a component of it

Reply:
also a bit of documentation for what these are would be nice